src/client/Health/NewNorth copy.jsx

```diff
@@ -0,0 +1,327 @@
+import React, { useState } from 'react'
+import { nanoid } from 'nanoid'
+import { Button, Col, Container, Row } from 'reactstrap'
+import { Modal } from 'react-bootstrap'
+import { ConfigContext } from '../context/configContext.jsx'
+import validationNorth from '../North/Form/North.validation'
+import { OIbText } from '../components/OIbForm'
+import iot from './iot.png'
+import database from './db-in.png'
+import file from './file-in.png'
+import Api from './api.png'
+import oi from './oi.png'
```

Comment:
Now you should use SVG. Moreover, can you rename the js variable with something unambiguous like 'iot' -> 'iotSvg', 'database' -> 'databaseSvg'. This will be clearer this way I think